fern/definition/medical/document.yml

```diff
@@ -102,7 +102,7 @@ service:
     getURL:
       display-name: Get Document URL
       docs: |
-        Gets a presigned URL for downloading the specified document.
+        Gets a downloadable URL for downloading the specified document.
```

Comment:
also here https://docs.metriport.com/medical-api/api-reference/document/get-document

we can put a <Tip> saying something like "For patients with a large volume documents, you can use the Bulk Download URL endpoint to get all download URLs, instead of doing it one at a time" or something like that